adapters/adtelligent/adtelligent.go

```diff
@@ -171,7 +171,8 @@ func validateImpression(imp *openrtb2.Imp) (int, error) {
 	// common extension for all impressions
 	var impExtBuffer []byte
 
-	impExtBuffer, err = json.Marshal(&adtelligentImpExt{
+	// TODO: confirm the missing error handling is intended
```

Comment:
The error handling is not intentionally skipped. All json.Marshal errors should be bubbled up as a fatal error.

Reply:
fixed in 146f9699